pkg/utils/utils_test.go

```diff
@@ -138,6 +138,29 @@ func TestIsUrl(t *testing.T) {
 	assert.Equal(t, IsURL("https://"), false)
 }
 
+func TestIsGitRepoUrl(t *testing.T) {
```

Comment:
In this test case, you need to add some more invalid test cases.

- oci url: oci://xxx/xxx
- file path on unix: /xxx/xxx/xxx or ./xxx/xxx/
- file path on windows: C:/xxx/xxx/xxx

Reply:
done!